.travis.yml

```diff
@@ -106,3 +151,4 @@ branches:
   - master
   - "/^.*release.*$/"
   - "/2\\.[0-9]+\\.[0-9]+/"
+  - "/^JSUI-[0-9]+/"
```

Comment:
`continuous-integration/travis-ci/push` builds are triggered by a Travis option on the repo that happened to be already selected, but they weren't triggered before due to the lack of this line.